commons/src/test/groovy/org.codice.ddf.admin.commons.requests/RequestUtilsTest.groovy

```diff
@@ -106,32 +113,38 @@ class RequestUtilsTest extends Specification {
         !report.containsFailureMessages()
         report.messages().find {it.type() == WARNING && it.subtype() == RequestUtils.UNTRUSTED_CA}
         report.messages().find {it.type() == SUCCESS && it.subtype() == RequestUtils.EXECUTED_REQUEST}
+
+        2 * utils.closeClientAndResponse(_, _)
```

Comment:
I'm not sure you need to be/should be this precise about the number of times these methods are expected to be called. Maybe a `(1.._)` range would be safer?